packages/generator/src/generators/app-generator.ts

```diff
@@ -15,10 +15,13 @@ export interface AppGeneratorOptions extends GeneratorOptions {
   version: string
   skipInstall: boolean
   skipGit: boolean
+}
+
+export interface AppGeneratorPromptOptions {
   form: "React Final Form" | "React Hook Form" | "Formik"
```

Comment:
suggestion: These can be extracted to an enum
``` 
enum FormOptions {
 FINAL_FORM = 'React Final Form',
...
}
```
Then can be referred to in other places by their enum:
```
    const formChoices: Array<{name: FormOptions; message?: string}> = [
      {name: FormOptions.FINAL_FORM, message: `${FormOptions.FINAL_FORM} (recommended)`},
    ]

    const formPrompt: Prompt = {
      name: "form",
      message: "Pick a form library (you can switch to something else later if you want)",
      choices: formChoices,
    }
```
```

Reply:
I'd rather just leave it as the discriminated union. It makes the error messages a bit more helpful (`expected one of "a" "b" or "c"` rather than `expected a FormOption`)